test/vm.install

```diff
@@ -15,7 +15,11 @@ if type firewall-cmd >/dev/null 2>&1; then
 fi
 
 # Install nodejs if it does not exist
-which node > /dev/null 2>&1 || yum install -y nodejs
+if grep -isq fedora /etc/redhat-release; then
+    command -v node > /dev/null 2>&1 || dnf install -y nodejs
+else
+    command -v node > /dev/null 2>&1 || yum install -y nodejs
+fi
```

Comment:
Could this rather test for `dnf` being available? RHEL 8 also has that. I. e.
```sh
command -v node > /dev/null 2>&1 || $(command -v dnf || command -v yum) install -y nodejs
```

Reply:
Better then `if`. :+1: Thanks @martinpitt.